src/scripts/lists.coffee

```diff
@@ -582,10 +589,10 @@ class ContentEdit.ListItemText extends ContentEdit.Text
 
                 # Convert the text item to a list item
                 cssClass = element.attr('class')
-                listItem = new ContentEdit.ListItem(
+                listItem = new element._factory.ListItem(
```

Comment:
This is crutch. Don't know how did it normal. If I using => this binds to origin class